src/tests/ftest/util/server_utils.py

```diff
@@ -122,30 +130,46 @@ def create_server_yaml(basepath):
         raise ServerFailed("Failed to Write {}/{}".format(basepath,
                                                           AVOCADO_FILE))
 
-def run_server(hostfile, setname, basepath, uri_path=None, env_dict=None):
-    """
-    Launches DAOS servers in accordance with the supplied hostfile.
+
+def run_server(hostfile, setname, basepath, uri_path=None, env_dict=None,
+               clean=True):
```

Comment:
Didn't notice anywhere in this patch that `clean` was set to `False`.  Did I just miss it somewhere?

Reply:
Line 273 in metadata.py

Reply:
Yup, did miss that.